dashboard/src/App.js

```diff
@@ -1,26 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
-//component imports
-import Index from "../src/components/Index";
+/*
+class App extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      items = []
+    }
+  }
+
+  render() {
+    const { items } = this.state;
+    return <>
+      <input> </input>
+      <h1> gg </h1>
+      <ul>
+        {items.map(item => <li>{item} </li>)}
+      </ul>
+    </>
+  }
+}*/
 
+//component imports
+import MainPage from "./components/MainPage";
+import Banner from "./components/Banner";
+import HowTo from "./components/HowTo";
+import Leaderboard from "./components/Leaderboard";
 //firebase imports
 
 //react + css
 
 //router 
 import {BrowserRouter as Router,Switch, Route } from "react-router-dom";
 
-function App() {
+const App = () => {
+
   return (
```

Comment:
since you're not doing any side effects, you can do () => ( <Component.... ) instead of brackets and return for slightly cleaner code